src/@next/Badge/BadgeStyle.tsx

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components';
+import { S08 } from '../utilities/colors/blue';
+import { B89 } from '../utilities/colors/green';
+import { B95 } from '../utilities/colors/neutral';
+import { S42, S87 } from '../utilities/colors/orange';
+import { B100 } from '../utilities/colors/red';
+import { S75 } from '../utilities/colors/yellow';
+import { space8 } from '../utilities/spacing';
+import { BadgeProps, BadgeStatusType } from './Badge';
+
+const badgeBackgroundColor = {
+  success: B89,
```

Comment:
it would be better to match the type with BadgeStatusType so we get an error if there's anything missing from the types. you'll have to do something [similar](https://github.com/glints-dev/glints-aries/blob/v4/src/%40next/Typography/TypographyStyles.ts#L179) with Typography to make it work and defined the BadgeStatusType differently.

Reply:
additional discussion:
i think we should put default type to status and on the props, if there’s no passed status, we default it to `status=default` that means if you map the default type to the status to color mapping, we’ll be able to handle it

Reply:
added default status value, but its `neutral` instead of `default` since i'm following sirius UI kit